### ISSUE TITLE
Problems installing

### ISSUE THREAD

**Issue by LSchueler** [malteschueler/skat_hunt_interface#1]
Okay, I followed the install instructions and got Windows 10 on an SSD running. But how do I quit WSappx?!
Thanks for the help

**Comment by malteschueler** [comment]
Have you tried googling it?

**Comment by LSchueler** [comment]
Yeah, all the help sites make the same stupid suggestions. GitHub now belongs to the same guys who created WSappx, right?! - Oh dear....
